catalog/nsili/catalog-nsili-common/src/main/java/org/codice/alliance/nsili/common/ResultDAGConverter.java

```diff
@@ -417,25 +450,54 @@ public static DAG convertResult(Result result, ORB orb, POA poa, List<String> re
 
         String attribute = parentAttrName + NsiliConstants.NSIL_SECURITY;
 
-        //TODO -- Once taxonomy is complete, set real values
         if (shouldAdd(buildAttr(attribute, NsiliConstants.CLASSIFICATION), resultAttributes)) {
-            addStringAttribute(graph,
-                    securityNode,
-                    NsiliConstants.CLASSIFICATION,
-                    NsiliClassification.UNCLASSIFIED.getSpecName(),
-                    orb);
-            addedAttributes.add(buildAttr(attribute, NsiliConstants.CLASSIFICATION));
+            Attribute metadataClassificationAttr = metacard.getAttribute(
+                    Security.METADATA_CLASSIFICATION);
+            if (metadataClassificationAttr != null) {
```

Comment:
If metadata classification is not defined, shouldn't we fall back to the overall classification (Security.CLASSIFICATION)?